fastapi/concurrency.py

```diff
@@ -2,9 +2,6 @@
 
 from starlette.concurrency import iterate_in_threadpool as iterate_in_threadpool  # noqa
 from starlette.concurrency import run_in_threadpool as run_in_threadpool  # noqa
-from starlette.concurrency import (  # noqa
```

Comment:
Would need to flagged as a breaking change in the change log? (message for tiangolo he makes the change logs)

Reply:
starlette walked back on the removal of this function